teamwork/templates/project_details.html

```diff
@@ -8,25 +8,26 @@
   <h1>{{ project.name }}</h1>
   <p>{{ project.description }}</p>
 
-  <!-- Group memebers -->
-  <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">
-        <i class="glyphicon glyphicon-user"></i>
-        Group member
-      </h3>
-    </div>
-    <div class="panel-body">
-      <ul class="list-group">
-        {% for member in project.members.all %}
-        <li class="list-group-item">{{ member.user.username }}</li>
-        {% endfor %}
-      </ul>
+  <!-- Group members -->
+        <p><i class="glyphicon glyphicon-user"></i>
+        Group member: {% for member in project.members.all %}
+        {{ member.user.username }}, 
+        {% endfor %}<p>
       <button type="button" class="btn btn-default" onclick="openEventCreator()">Add member</button>
-    </div>
-  </div>
 
   <!-- Task list -->
+  <h3>Tasks List</h3>
+  <ul class="list-group">
+    {% for task in project.task_set.all %}
+    <li class="list-group-item">
+      <input type="checkbox" 
+        {% if task.done is True %}

```

Comment:
Because `task.done` evaluates to boolean, `if task.done is True` can be replaced with `if task.done`